src/Templating/src/Microsoft.DotNet.Web.ProjectTemplates/content/RazorComponentsWeb-CSharp/.template.config/template.json

```diff
@@ -5,13 +5,16 @@
     "Web",
     "RazorComponents"
   ],
-  "name": "ASP.NET Core Razor Components",
+  "name": "ASP.NET Core Web App (Razor Components)",
   "generatorVersions": "[1.0.0.0-*)",
-  "description": "A project template for creating an ASP.NET Core Razor Components application. This template can be used for web applications with rich dynamic user interfaces (UIs).",
+  "description": "A project template for creating an ASP.NET Core application with example ASP.NET Core Razor Components. This template can be used for web applications with rich dynamic user interfaces (UIs).",
```

Comment:
I'd remove the word `example` from here as I think it would be clearer, but if the existing phrasing was already debated at length then either way is fine.

Reply:
Also it's odd to repeat `Razor Components` twice within a short span of words. Suggested phrasing:

```
A project template for creating an ASP.NET Core application with Razor Components. This template can be used for web applications with rich dynamic user interfaces (UIs).
```

CC @danroth27